### PR TITLE
Update credits

### PR DESCRIPTION
**Pull request by Zeno-**

### CODE REVIEW

**Comment by sofart** [comment]
:+1:

**Comment by paramat** [comment]
^ Who is that? @sofar

**Comment by nerzhul** [comment]
wtf

**Comment by paramat** [comment]
sofart's comment disappeared.

**Comment by Zeno-** [comment]
This is currently a "placemarker PR". Essentially to remind us to update this file before release.

**Comment by red-001** [comment]
@paramat I guess GitHub support might have responded to my report then.

File: builtin/mainmenu/tab_credits.lua [hunk 1]
```diff
@@ -38,26 +38,26 @@ local active_contributors = {
...
 	"Jeija <jeija@mesecons.net>",
-	"Gregory Currie (gregorycu)",
-	"Sokomine <wegwerf@anarres.dyndns.org>",
 	"TeTpaAka",
 	"Jean-Patrick G (kilbith) <jeanpatrick.guerrero@gmail.com>",
 	"Diego Martínez (kaeza) <kaeza@users.sf.net>",
 	"Dániel Juhász (juhdanad) <juhdanad@gmail.com>",
 	"Rogier <rogier777@gmail.com>",
+	"red-001 <red-001@outlook.ie>",
+	"Vanessa Ezekowitz (VanessaE) <vanessaezekowitz@gmail.com>",
```
Comment (nerzhul) [inline builtin/mainmenu/tab_credits.lua]: no amount of PR in the core engine, should be removed

File: builtin/mainmenu/tab_credits.lua [hunk 2]
```diff
@@ -28,7 +28,7 @@ local core_developers = {
 	"Loic Blot (nerzhul/nrz) <loic.blot@unix-experience.fr>",
 	"Matt Gregory (paramat)",
 	"est31 <MTest31@outlook.com>",
-	"Craig Robbins (Zeno) <craig.d.robbins@gmail.com>",
+	"Craig Robbins (Zeno) <craig.d.robbins@gmail.com> (Listens to ABBA)",
```
Comment (nerzhul) [inline builtin/mainmenu/tab_credits.lua]: don't forget to remove this :)

**Comment by Fixer-007** [comment]
BTW, I've seen conversation on #minetest-dev on who to include or not.
"About" tab seems to me too much complicated, maybe instead: 
+ make just two categories: Engine and Minetest Game (without split of active/nonactive, you can start with active contributors on top)
+ define criteria. for example amount of code added/changed/commits/etc, so you can define major contributors in meaningful way
+ list them, in both categories at the end add "end other contributors"
+ in the end of about tab add separate sentence "Thanks to translators who participated in the project", you can even make third category just for them, listing active language translators and their languages.

**Comment by red-001** [comment]
For translators maybe just credit the top translators of the language the player is playing in or something like that

File: builtin/mainmenu/tab_credits.lua [hunk 3]
```diff
@@ -28,7 +28,7 @@ local core_developers = {
 	"Loic Blot (nerzhul/nrz) <loic.blot@unix-experience.fr>",
 	"Matt Gregory (paramat)",
```
Comment (paramat) [inline builtin/mainmenu/tab_credits.lua]: Just 'paramat' please, lower case 'p'.

**Comment by paramat** [comment]
Including MTGame contributors may not be a good idea, it's a separate project theoretically no different to any other subgame.

**Comment by Fixer-007** [comment]
Then subgame should have some kind of description info via some button with list of mods and contributors.

**Comment by paramat** [comment]
Another suggestion to consider: In each release list all those who have commits in the release. This would restrict numbers but also mean we don't have to draw a line at some arbitrary number of commits.
But 'previous contributors' would have to remain a restricted number to avoid getting too big.

File: builtin/mainmenu/tab_credits.lua (hunk 1 above)
Comment (Zeno-) [inline builtin/mainmenu/tab_credits.lua]: Quite often there are PRs that need live testing before they are added to the engine. Often the person who does this is Vanessa, on her actual servers (not "test" servers). To me that is contributing and a valuable one. I don't think Vanessa cares if her name is in this section or not, but I believe it does belong here. Developing is about more than just coding.

File: builtin/mainmenu/tab_credits.lua (hunk 3 above)
Comment (Zeno-) [inline builtin/mainmenu/tab_credits.lua]: Just clarifying: you're asking for your real name to be removed?

File: builtin/mainmenu/tab_credits.lua (hunk 2 above)
Comment (Zeno-) [inline builtin/mainmenu/tab_credits.lua]: I hope I don't forget :D

**Comment by est31** [comment]
@Zeno- has asked me via IRC whether I want something changed about my entry.

The current entry is okay.

File: builtin/mainmenu/tab_credits.lua [hunk 4]
```diff
@@ -26,38 +26,39 @@ local core_developers = {
 	"ShadowNinja <shadowninja@minetest.net>",
 	"Nathanaël Courant (Nore/Ekdohibs) <nore@mesecons.net>",
 	"Loic Blot (nerzhul/nrz) <loic.blot@unix-experience.fr>",
-	"Matt Gregory (paramat)",
-	"est31 <MTest31@outlook.com>",
-	"Craig Robbins (Zeno) <craig.d.robbins@gmail.com>",
+	"paramat",
+	"Craig Robbins (Zeno) <craig.d.robbins@gmail.com> (Dancing Queen)",
```
Comment (SmallJoker) [inline builtin/mainmenu/tab_credits.lua]: Untrue statement. You might feel like a queen and dance but this doesn't make you to a queen.

Comment (DS-Minetest) [inline builtin/mainmenu/tab_credits.lua]: I suggest `Dancing Person feeling like a Queen` or `Dancing Pseudoqueen` or `Dancing Quasiqueen`.

**Comment by Zeno-** [comment]
Too controversial for something this trivial. Closing.

**Comment by paramat** [comment]
Most of this is good and needed, adoption needed.

**Comment by paramat** [comment]
How about this to reduce arguments over inclusion:
"Another suggestion to consider: In each release list all those who have commits in the release. This would restrict numbers but also mean we don't have to draw a line at some arbitrary number of commits.
But 'previous contributors' would have to remain a restricted number to avoid getting too big."

**Comment by nerzhul** [comment]
paramat see #5840

**Pull request closed without merging**